src/lib/comparison.js

```diff
@@ -67,6 +67,20 @@ function compareIn(field, values, normaliseValues) {
   return joinRules(compareMap(field, 'eq', values, normaliseValues), 'or');
 }
 
+function compareAll(objectValue, normaliseValues) {
+  const keys = Object.keys(objectValue);
+  return joinRules(keys.map(k => {
+    const ov = objectValue[k];
```

Comment:
mb use `field` and `value` names? To make it similar as in other "operators"
```suggestion
    const value = objectValues[field];
```

Reply:
updated :)